src/extension.ts

```diff
@@ -90,7 +90,7 @@ function whenOpeningTextDocument(
     return;
   }
 
-  const folderPath = folder.uri.toString();
+  const folderPath = folder.uri.path;
```

Comment:
Does this change give us anything? I feel like we shouldn't use unix-only path if there's not a clear reason; If we should, then we ideally need to somehow document the rationale as well

Reply:
I think it was just to make hashing identical across systems. I can't remember if it was actually necessary (probably not, so I'll remove it)

Reply:
probably not, I think it was just to make hashing identical across systems.